tests/kernel/blktests.pm

```diff
@@ -29,6 +29,9 @@ sub run {
     my $config  = get_required_var('BLK_CONFIG');
     my $device  = get_required_var('BLK_DEVICE_ONLY');
 
+    #install test specific tools
+    zypper_call('in fio blktrace');
```

Comment:
This will not help at all.qa_test_fio contains `Provides:       fio`.

Reply:
this did help. Test are run. Please see runs.

Reply:
It also provides `Obsoletes:      fio`, this is probably reason why it is selected as dependency instead of `fio`.